integrations/amazon_bedrock/tests/test_chat_generator.py

```diff
@@ -430,6 +445,45 @@ def test_live_run_with_multi_tool_calls_streaming(self, model_name, tools):
         assert "paris" in final_message.text.lower()
         assert "berlin" in final_message.text.lower()
 
+    @pytest.mark.parametrize("model_name", [STREAMING_TOOL_MODELS[0]])  # just one model is enough
+    def test_live_run_with_tool_with_no_args_streaming(self, tool_with_no_parameters, model_name):
+        """
+        Integration test that the AmazonBedrockChatGenerator component can run with a tool that has no arguments and
+        streaming.
+        """
+        initial_messages = [ChatMessage.from_user("Print Hello World using the print hello world tool.")]
+        component = AmazonBedrockChatGenerator(
+            model=model_name, tools=[tool_with_no_parameters], streaming_callback=print_streaming_chunk
+        )
+        results = component.run(messages=initial_messages)
+
+        assert len(results["replies"]) == 1
+        message = results["replies"][0]
+
+        # # this is Claude thinking message prior to tool call
+        assert message.text is not None
+
+        # now we have the tool call
+        assert message.tool_calls
+        tool_call = message.tool_call
+        assert isinstance(tool_call, ToolCall)
+        assert tool_call.id is not None
+        assert tool_call.tool_name == "hello_world"
+        assert tool_call.arguments == {}
+        assert message.meta["finish_reason"] == "tool_use"
+
+        # new_messages = [
+        #     *initial_messages,
+        #     message,
+        #     ChatMessage.from_tool(tool_result="Hello World!", origin=tool_call),
+        # ]
+        # results = component.run(new_messages)
+        # assert len(results["replies"]) == 1
+        # final_message = results["replies"][0]
+        # assert not final_message.tool_calls
+        # assert len(final_message.text) > 0
+        # assert "hello" in final_message.text.lower()
```

Comment:
feel free to delete this if not necessary for the test

Reply:
ah, no I temporarily commented these lines to generate `StreamingChunk` for the unit test and I forgot to uncomment them.
Now it's fixed